plugin/pom.xml

```diff
@@ -22,24 +22,12 @@
         <dependency>
             <groupId>io.jenkins.plugins.remoting-kafka</groupId>
             <artifactId>kafka-client-lib</artifactId>
-            <version>1.1-SNAPSHOT</version>
+            <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
             <version>2.1.17</version>
         </dependency>
     </dependencies>
-    <repositories>
-        <repository>
-            <id>repo.jenkins-ci.org</id>
-            <url>https://repo.jenkins-ci.org/public/</url>
-        </repository>
-    </repositories>
-    <pluginRepositories>
-        <pluginRepository>
-            <id>repo.jenkins-ci.org</id>
-            <url>https://repo.jenkins-ci.org/public/</url>
-        </pluginRepository>
-    </pluginRepositories>
```

Comment:
We need these tags in plugin/ to do the release in Jenkins update center.

Reply:
They should already exist in the parent POM

Reply:
Tbh I think it would be better to move plugin to separate repo so we could support incrementals for both plugin and the agent and kafka-client-lib modules.

Reply:
Oh keep the plugin here since that is the name of the repo and then move the other modules to a sparate repo 😅

Reply:
It could be doable, of course. But maybe it is an overkill for now. A follow-up task would be useful

Reply:
@casz These tags are needed because the `plugin/` component doesn't use parent POM. Hence, it is not possible to release without these tags.

Reply:
I'll just close the PR then since the version stuff you can do separately

Reply:
@casz you can revert these tags and we can integrate the version change.

Reply:
Will do